### PR TITLE
Preparing for 8.0

### LINKED ISSUE #798: General 8.0 Release Checklist
- [x] s/7\\.5/8.0/g
  - `find . -name '*.rst' -exec grep -Hn '\(\b7\.\?[1-5]\b\)\|\(7[1-5]_\)' {} \;`
- [x] Update all installation steps to use new package name (except CloudFormation)
- [x] Update links to new CloudFormation template
  - [x] Test deploying CF template (check citus_version)
    - All good except it does use postgres 9.6.10 which is slightly old
  - [x] curl the urldecoded templateURL as anonymous user to verify public permissions
- [x] Update docs to use pg 11 rather than 10
- [x] Verify Installation steps with new packages (single-machine cluster install)
    - [x] Ubuntu 18.04
    - [x] Ubuntu 16.04 (Joe)
    - [x] Debian Stretch
    - [x] Debian Jessie
    - [x] CentOS 7.5
    - [x] CentOS 6.9
    - [x] RHEL 7.5
    - [x] RHEL 6.9
- [x] Update our upgrade steps
    - [x] Test 7.5->8.0 major upgrade on pg 10, ubuntu
    - [x] Test 8.0/10 -> 8.0/11
- [x] Test Multi-tenant tutorial and use-case guide (on just one version)
- [x] Test Real-time analytics tutorial and use-case guide (on Docker)
- [x] Verify there are no Sphinx warnings in local build
    - (Remove _build directory and try fresh too)
- [x] Update citusdata.com/download links. (https://github.com/citusdata/citusdata.com/pull/256)
  - [x] Confirm with Claire that it goes live after merge
- [x] Create new 8.0 release branch and set as default ver in RTD
- [x] Verify that RTD adds functioning analytics code
- [ ] Verify heap and segment
- [x] Rebuild previous version so that canonical link gets updated
- [x] Duplicate RTD page redirects from previous to new version
- [x] Add redirect for MX leaving the cloud section

### PR DESCRIPTION
**Pull request by begriffs**
Fixes #798 

We'll probably want to rebase this on top of the what is citus changes and new 8.0 features pr once they are both merged. That way I can check that there are still no sphinx warnings when it all gets put together.

### CODE REVIEW

**Comment by begriffs** [comment]
The checklist is complete.

File: admin_guide/upgrading_citus.rst [hunk 1]
```diff
@@ -15,22 +15,22 @@ Upgrading the Citus version requires first obtaining the new Citus extension and
 Patch Version Upgrade
 ---------------------
 
-To upgrade a Citus version to its latest patch, issue a standard upgrade command for your package manager. Assuming version 7.5 is currently installed:
+To upgrade a Citus version to its latest patch, issue a standard upgrade command for your package manager. Assuming version 8.0 is currently installed:
```
Comment (sumedhpathak) [inline admin_guide/upgrading_citus.rst]: Maybe add "Citus version 8.0 is installed with PostgreSQL 11".

File: admin_guide/upgrading_citus.rst [hunk 2]
```diff
@@ -39,7 +39,7 @@ Major and Minor Version Upgrades
 
 Major and minor version upgrades follow the same steps, but be careful: major upgrades can make backward-incompatible changes in the Citus API. It is best to review the Citus `changelog <https://github.com/citusdata/citus/blob/master/CHANGELOG.md>`_ before a major upgrade and look for any changes which may cause problems for your application.
 
-Each major and minor version of Citus is published as a package with a separate name. Installing a newer package will automatically remove the older version. Here is how to upgrade from 6.2 to 7.5 for instance:
+Each major and minor version of Citus is published as a package with a separate name. Installing a newer package will automatically remove the older version. Here is how to upgrade from 7.5 to 8.0 for instance:
```
Comment (sumedhpathak) [inline admin_guide/upgrading_citus.rst]: This needs to change. I think here we need to clarify that 7.5 won't work with PG11.

File: admin_guide/upgrading_citus.rst [hunk 3]
```diff
@@ -49,16 +49,16 @@ Step 1. Update Citus Package
...
   sudo service postgresql restart
 
 **Fedora, CentOS, or Red Hat**
 
 .. code-block:: bash
 
   # Fedora, CentOS, or Red Hat
-  sudo yum swap citus62_96 citus75_96
-  sudo service postgresql-9.6 restart
+  sudo yum swap citus75_11 citus80_11
```
Comment (sumedhpathak) [inline admin_guide/upgrading_citus.rst]: We need to replace 11 with 10 here. An upgrade from 7.5 to 8.0 will need to be on PG10. There aren't packages generated for 7.5 and 11. So `citus75_11` doesn't exist and will error out.

File: admin_guide/upgrading_citus.rst (hunk 2 above)
Comment (sumedhpathak) [reply admin_guide/upgrading_citus.rst]: We also need to replace 11 with 10 everywhere here. Any update from 7.5 to 8.0 will need to be first on PG10.

File: admin_guide/upgrading_citus.rst [hunk 4]
```diff
@@ -88,22 +88,26 @@ After installing the new package and restarting the database, run the extension
...
 .. _upgrading_postgres:
 
-Upgrading PostgreSQL version from 9.6 to 10.0
-#############################################
+Upgrading PostgreSQL version from 10 to 11
+##########################################
+
+.. note::
+
+   Do not attempt to upgrade *both* Citus and Postgres versions at once. If both upgrades are desired, start with a database upgrade first.
```
Comment (sumedhpathak) [inline admin_guide/upgrading_citus.rst]: This is a bit ambiguous. What do we mean by 'database upgrade'? I think we should note that 7.5 *does not* work with PG11. So any update to 11 *requires* a Citus major version upgrade to 8.0. Only then can we upgrade to PG11.

File: develop/migration_rt.rst [hunk 5]
```diff
@@ -1,6 +0,0 @@
-Real-Time Analytics Apps
```
Comment (sumedhpathak) [inline develop/migration_rt.rst]: Why is this deleted?

File: admin_guide/upgrading_citus.rst (hunk 2 above)
Comment (begriffs) [reply admin_guide/upgrading_citus.rst]: I left pg 11 in the "patch update" section where it's assuming an upgrade within citus 8, but switched to 10 for anything involving 7.5

File: develop/migration_rt.rst (hunk 5 above)
Comment (begriffs) [reply develop/migration_rt.rst]: Part of the release checklist is to fix any sphinx build warnings. This file wasn't referenced in any menu or document, so sphinx was complaining and I removed the file.

If you prefer we can restore it and add a menu entry.

File: admin_guide/upgrading_citus.rst [hunk 6]
```diff
@@ -88,22 +88,26 @@ After installing the new package and restarting the database, run the extension
...
 .. _upgrading_postgres:
 
-Upgrading PostgreSQL version from 9.6 to 10.0
-#############################################
+Upgrading PostgreSQL version from 10 to 11
+##########################################
+
+.. note::
+
+   Do not attempt to upgrade *both* Citus and Postgres versions at once. If both upgrades are desired, upgrade Citus first. Older versions of Citus are not always compatible with the newest Postgres.
```
Comment (sumedhpathak) [inline admin_guide/upgrading_citus.rst]: Can we add some more specifics here? e.g.: Citus 7.x is *not* compatible with PG11. So to upgrade PG 10 to 11, follow the above steps <link to Citus major version upgrade> to first upgrade from Citus 7.x to 8.0. Once you have done so, then continue below.

File: develop/migration_rt.rst (hunk 5 above)
Comment (sumedhpathak) [reply develop/migration_rt.rst]: I didn't realize this was not being displayed anywhere. Did this disappear as part of our migration section revamp? I thought it was a small blurb in that section initially. See: http://docs.citusdata.com/en/v7.3/develop/migration.html.

Was this consciously removed, or just missed? The content is pretty content-free, so I'm OK removing, but have some note to contact-us (or maybe links to our use-case guides / customer case-studies) might be helpful.

Comment (begriffs) [reply develop/migration_rt.rst]: > Did this disappear as part of our migration section revamp?

Yeah, it didn't fit in with the nice progression of MT steps, and it didn't have much information. The only reason to keep it in is to clarify that the rest of the migration content is skewed towards the MT case (the section is labeled as just "migrating and existing app" which appears more general than it is)

Comment (sumedhpathak) [reply develop/migration_rt.rst]: Ok, then could you open up a new issue to upgrade our content? Could you also capture this deleted information in that issue (Just as context for what we used to have)?

We can discuss it outside of the 8.0 release. Thanks
